jwala-services/src/main/java/com/cerner/jwala/service/app/impl/ApplicationServiceImpl.java

```diff
@@ -130,20 +142,39 @@ public Application getApplication(final String name) {
 
     @Transactional
     @Override
-    public Application updateApplication(UpdateApplicationRequest updateApplicationRequest, User anUpdatingUser) {
+    public Application updateApplication(UpdateApplicationRequest updateApplicationRequest, User anUpdatingUser)
+            throws Exception{
         updateApplicationRequest.validate();
-
         final Application application = applicationPersistenceService.updateApplication(updateApplicationRequest);
+
+        String appName = application.getName();
+        Identifier<Group> newGroupId = application.getGroup().getId();
+        Long id = newGroupId.getId();
+
+        List<Long> idList = new ArrayList<>();
+        idList.add(id);
```

Comment:
You can create the list and populate at the same time with a single line of code using Collections.singletonList()